common/helpers.js

```diff
@@ -86,6 +98,14 @@ function dataObjectUriToHttps(dataObjectUri) {
     return output;
 }
 
+// This function counts on the request posing  data as "application/json" content-type.
```

Comment:
nit: there are two spaces between "posing" and "data"

Reply:
removed the extra space 👍